src/main/java/io/github/bbortt/event/planner/repository/UserRepository.java

```diff
@@ -31,11 +30,11 @@
 
     Optional<User> findOneByLogin(String login);
 
-    @EntityGraph(attributePaths = "authorities")
+    @EntityGraph(attributePaths = { "authorities", "invitations" })
```

Comment:
Nur eine Frage: Was passiert, wenn man hier etwas falsches angibt? Wirft die Applikation dann gleich beim Starten? Oder merkt man das erst, wenn man darauf zugreift?
@bbortt

Reply:
dies wird beim Startup nicht bemerkt. allerdings wird die Funktion auch in diversen Integration-Tests verwendet, z.B. `AccountResourceIT`.